projects/pgai/.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
If you don't use a `.python-version` file, uv will take the newest python version that's compatible with your pyproject.toml. I don't think this is desirable for development as it makes us potentially unaware that we are using newer python features than we want. So explicitly setting this here.

Reply:
`.python-version` files support comments, can we add one here explaining why we use 3.10?

```suggestion
# the 'timescaledb-ha' docker image uses 3.10, so do we
3.10
```

Reply:
The pgai library doesn't run in the timescaledb-ha image. So, we may not need this constraint.

Reply:
added a comment for the actual reason :)

Reply:
Hmm the comments do not actually seem to be allowed:
```
  Error: The version '3.10
  # We try to somewhat follow the python release schedule for supported versions to allow more users to use our library.
  # Therefore we are about a year behind the latest python release.' with architecture 'x64' was not found for Ubuntu 22.04.
  ```
  
  Adding this to the dev docs instead.

Reply:
Hmmm. interesting. I did try out comments with uv locally before making this comment - maybe it was something that was added relatively recently.